### ISSUE TITLE
Home button tooltip

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#3883]
The tooltip for the home is "Canada". This is wrong because initial extent maybe something else. Should be "Initial extent" instead

**Comment by jolevesq** [comment]
@james-rae As part of the next release where some left over WCAG bugs may be fix, can we add this?

**Comment by james-rae** [comment]
It's a big request but we can try to swing it 😄 .

How do you feel about "Initial View"?  "Extent" might be a bit technical for the average person.

Also if you can provide us with a French translation that seems appropriate to you that would help.

**Comment by jolevesq** [comment]
You are right, view is better then extent.

We can say: Initial view - Vue initiale

Thanks
